sdk/cosmos/azure-cosmos-table/src/main/java/com/microsoft/azure/tables/AzureTable.java

```diff
@@ -0,0 +1,108 @@
+package com.microsoft.azure.tables;
```

Comment:
1. The MIT header license should have been generated on the top of each of these files.
  1. If you add `--license-header:MICROSOFT_MIT_SMALL` when generating these classes, it'll add the license header to the top.
1. The package name should be `com.azure.cosmos.table` (table singular because your folder name is singular and I asserted that you meant for the package name to be singular, too).
  1. https://azure.github.io/azure-sdk/java_design.html#example-namespaces